native/apache-2.0/Makefile.vc

```diff
@@ -16,38 +16,25 @@
 PROJECT = mod_jk
 # Tools
 CC = cl.exe
-LINK = link.exe
+LN = link.exe
 RC = rc.exe
-MT = mt.exe
 
 !IF !DEFINED(APACHE2_HOME) || "$(APACHE2_HOME)" == ""
 !ERROR Cannot find APACHE2_HOME. Use nmake -f Makefile.vc APACHE2_HOME=dir
 !ENDIF
 !IF !DEFINED(APACHE2_LDIR) || "$(APACHE2_LDIR)" == ""
-APACHE2_LDIR=$(APACHE2_HOME)\lib
+APACHE2_LDIR = lib
```

Comment:
It looks like this should not change. If the admin wants to change the lib (or include, etc.) dirs, they cannot now because they will always have to be relative to `APACHE2_HOME`. Without this change, a full path can be specified for the `APACHE2_LIB` (`_INCLUDE`, etc.) including a drive letter, but this change makes that impossible.

I don't know if there are any use-cases for specifying different lib, include, etc. directories but this looks like a loss in functionality.

Reply:
My intention was to simplify the things by not requiring the full path for lib and include dirs,
since APACHE2_HOME should point to the directory containing both lib and include/httpd and include/apr-1.
The APACHE2_IDIR makes no sence, since it should always be $(APACHE2_HOME)\include.
Same for APACHE2_LDIR that is by default $(APACHE2_HOME)\lib.
However one can set APACHE2_LDIR=lib64, which would evaluate to $(APACHE2_HOME)\lib64